util/dvsim/utils.py

```diff
@@ -103,7 +103,8 @@ def _stringify_wildcard_value(value):
     try:
         return ' '.join(_stringify_wildcard_value(x) for x in value)
     except TypeError:
-        raise ValueError('Wildcard had value {!r} which is not of a supported type.')
+        raise ValueError('Wildcard had value \"{!r}\" which is not of a '
+                         'supported type.'.format(value))
```

Comment:
Hmm, the problem with this is that it now always looks like a string. The advantage of `repr` is that the output is unambiguous: I think that adding quotes probably makes it more confusing, not less.

Reply:
Removed.